classes/classes/Creature.as

```diff
@@ -381,7 +381,7 @@ package classes
 		//MALE STUFF
 		//public var cocks:Array;
 		//TODO: Tuck away into Male genital class?
-		public var cocks:Array;
+		public var cocks:/*Cock*/Array;
```

Comment:
The advantage (and in Actionscript with a gotcha) is that Vectors are typed, whereas arrays are not.
**However** Actionscript being retarded as it is, will only allow *that exact class* into a vector, *subclasses cannot be inserted into a vector of a super type.*

[AS3 Vector API](http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/Vector.html) see the second code example.